merlin/data/dataorganization.py

```diff
@@ -31,40 +33,56 @@ class DataOrganization(object):
     image files.
     """
 
-    def __init__(self, dataSet, filePath: str = None):
+    def __init__(self, dataSet, filePath: str = None,
+                 dataPortal: dataportal.DataPortal = None):
         """
         Create a new DataOrganization for the data in the specified data set.
 
-        If filePath is not specified, a previously stored DataOrganization
-        is loaded from the dataSet if it exists. If filePath is specified,
-        the DataOrganization at the specified filePath is loaded and
-        stored in the dataSet, overwriting any previously stored
-        DataOrganization.
+        The DataOrganization is located in the following search order:
+        i) If filePath is specified and filePath exists this file is copied
```

Comment:
Could you change this to indicate to the user that it first looks for the filepath, if it doesn't find that it falls back to looking for the filepath prepended with merlin.DATA_ORGANIZATION_HOME. It also seems like we should issue a warning or error to the user if they have something in filepath that doesn't exist in either location, in case it is able to get a dataorganiation file from one of the other two if statements.